docker/README.md

```diff
@@ -1,8 +1,5 @@
 # Docker
 
-This directory contains [Docker](https://www.docker.com/) configuration and scripts for building
-images of the validator node, faucet server, and client.
-
-Each subdirectory contains a build.sh to build the respective Docker container.
+This directory contains [Docker](https://www.docker.com/) configuration for building Aptos docker images.
```

Comment:
Could you please add more description here to explain the files in this repo and how people should build docker locally if needed?

Reply:
I think this is somewhat out of scope of this PR which just aims to just cleanup the old code.
I added a follow up ticket on #1412 . I'll work on this in conjunction with supporting M1 local builds.

Reply:
I'm screenshotting this promise.

Reply:
![image](https://user-images.githubusercontent.com/73818/173754921-6bc347fa-08f3-4c81-a02d-102b0d48fcb1.png)